src/Facebook/HttpClients/FacebookCurl.php

```diff
@@ -33,9 +33,9 @@ class FacebookCurl
 {
 
   /**
-   * @var resource Curl resource instance
+   * @var resource|null Curl resource instance
    */
-  protected $curl = null;
```

Comment:
There was a recent PR that added this. You'll need to duke it out with the original PR. :) See: #328 & #327.